### DISCUSSION TITLE
Problemi Webhooks

### DISCUSSION THREAD

**Discussion by paonza** [fattureincloud/api#413] [Q&A] [answered]
Buongiorno,

da qualche giorno non ricevo piu' le notifiche webhooks... hanno funzionato benissimo fino al 10 febbraio
dove ho ricevuto l'ultima: 328:XXXXXXXJafKlTbLLJPD9MFr

successivamente ho fatto diverse operazioni che avrebbero dovuto generare un trigger per le notifiche ma non sono arrivate.

Ho anche cancellato l'utlima sottoscrizione e ricreata:

Array
(
    [data] => Array
        (
            [id] => SUB531
            [sink] => https://www.XXXXXXXXXXX.net/fic_webhooks.php
            [verified] => 
            [types] => Array
                (
                    [0] => it.fattureincloud.webhooks.issued_documents.invoices.create
                    [1] => it.fattureincloud.webhooks.issued_documents.orders.create
                    [2] => it.fattureincloud.webhooks.issued_documents.invoices.update
                    [3] => it.fattureincloud.webhooks.issued_documents.invoices.delete
                )

            [config] => Array
                (
                    [mapping] => structured
                )

        )

    [warnings] => 
)

purtroppo non ho ricevuto nemmeno il welcome message per attivare la verifica.

Potete controllare per piacere, senza webhooks, le funzioinalita' del portale sono molto limitate!

grazie per l'aiuto,
saluti
Fabio

**Comment by emajo** [comment, marked as answer]
Ciao @paonza, ci risulta che la catena dei webhook sia funzionante, ti chiedo di scrivere al nostro supporto clienti così possiamo andare a fare controlli più specifici sull'azienda con cui stai avendo problemi.

Grazie,
Emanuele.

**Reply by paonza** [reply]
ho scritto a info@fattureincloud.it, sembra quasi che i webhooks siano stati staccati per il company ID... non so... andavano perfettamente, io loggo tutto quello che mi arriva e al momento il log sono vuoti...

**Reply by valmoz** [reply]
buongiorno @paonza il problema è stato risolto?

grazie
Mauro
